app/_data/docs_nav_kuma_dev.yml

```diff
@@ -805,3 +807,150 @@ items:
             url: "/contribute/introduction/#gui-code"
           - text: Testing unreleased versions
             url: "/contribute/introduction/#testing-unreleased-versions"
+unlisted:
+  - url: /
+  - url: /documentation/dependencies/
+  - url: /documentation/introduction/
```

Comment:
I think this page can be removed (it was likely just inherited from the past)

Reply:
Loos like this one is sill referenced [here](https://github.com/kumahq/kuma-website/blob/master/app/docs/dev/documentation/configuration.md?plain=1#L101)

Reply:
Can we just rewrite this line to:

When {{site.mesh_product_name}} (`kuma-cp`) is up and running it needs to store its state on Universal it's Postgres, for Kubernetes it's leveraging Kubernetes custom resource definitions.

Please :)